src/network-layer/default/RelayDefaultNetworkLayer.js

```diff
@@ -121,17 +121,22 @@ class RelayDefaultNetworkLayer {
         }
       }
       init = {
+        ...this._init,
         body: formData,
         method: 'POST',
       };
     } else {
       init = {
+        credentials: 'same-origin',
```

Comment:
Does this still make sense as a default? Or should it instead be pre-configured in [`src/__forks__/Relay.js`](https://github.com/facebook/relay/blob/master/src/__forks__/Relay.js#L20)?